client/src/lib/firebase/game.ts

```diff
@@ -0,0 +1,12 @@
+import { doc, updateDoc } from "firebase/firestore";
+import { lobbyCollection } from "$lib/firebase/firestore-collections";
+import type { GameState } from "./firestore-types/lobby";
+
+/*
+  You can change the game state of the current lobby
+*/
+export const changeGameState = async (lobbyId: string, state: GameState) => {
+  await updateDoc(doc(lobbyCollection, lobbyId), {
+    state,
+  });
+};
```

Comment:
I feel like for security reasons we might want this to be on the server but we can fix this as part of [#30](https://github.com/cat-acronym-team/cat-me-if-you-can/issues/30)